README.md

```diff
@@ -12,27 +12,40 @@ npm i lightstep-opentelemetry-launcher-node
 
 ### Configure
 
-Minimal setup
+#### Setup Tracing
+
+Put the following in `tracing.js`
 
 ```javascript
-const {
-  lightstep,
-  opentelemetry,
-} = require('lightstep-opentelemetry-launcher-node');
+const { lightstep } = require('lightstep-opentelemetry-launcher-node');
 
 const sdk = lightstep.configureOpenTelemetry({
   accessToken: 'YOUR ACCESS TOKEN',
   serviceName: 'otel-example',
 });
 
-sdk.start().then(() => {
-  // Make sure to load any modules you use after otel is started so that it
-  // has its module loading hooks in place. In general, this is the right place
-  // to require your code.
-  require('./server-main.js').startServer();
+// initialize and start the SDK
+sdk.start();
+
+// Gracefully shutdown the SDK
+const process = require("process");
```

Comment:
minor consistency: using single quotes at the top of the example and double quotes here